packages/performance/src/services/perf_logger.test.ts

```diff
@@ -40,6 +40,12 @@ describe('Performance Monitoring > perf_logger', () => {
   const TRACE_NAME = 'testTrace';
   const START_TIME = 12345;
   const DURATION = 321;
+  // Perf event header which is constant across tests in this file.
+  const webAppInfo = `"application_info":{"google_app_id":"${APP_ID}",\
```

Comment:
File level constants should be capitalized.

Reply:
Done. Modified it to be WEBAPP_INFO.